test/cppsuite/test_harness/workload/database_operation.h

```diff
@@ -363,7 +365,7 @@ class database_operation {
       int64_t value_size)
     {
         WT_DECL_RET;
-        WT_CURSOR *cursor;
+        WT_CURSOR *cursor, *op_track_cursor = nullptr;
```

Comment:
You can take the opportunity to make cursor a nullptr too

Reply:
The only reason I've set it to `nullptr` is because it's going to be unset when workload tracking is disabled. That doesn't apply to `cursor`.